extension-config.json

```diff
@@ -10,6 +10,24 @@
         "type": "json" ,
         "label": "Map for sort options"
       }
+    },
+    "defaultSortOption": {
```

Comment:
fyi in CURB we have already a setting for the default sort order.

Reply:
Sounds nice